components/Hume/Expressions.tsx

```
@@ -0,0 +1,54 @@
+"use client";
+import { expressionColors, isExpressionColor } from "@/utils/HumeAi/expressionColors";
+import { expressionLabels } from "@/utils/HumeAi/expressionLabels";
+import { motion } from "framer-motion";
+import { CSSProperties, use, useMemo } from "react";
```

ESLint

[error] 'use' is defined but never used.

(@typescript-eslint/no-unused-vars)